src/rest.cpp

```diff
@@ -353,7 +353,8 @@ static bool rest_tx(HTTPRequest* req, const std::string& strURIPart)
 
     CTransaction tx;
     uint256 hashBlock = uint256();
-    if (!GetTransaction(hash, tx, Params().GetConsensus(), hashBlock, true))
+    uint32_t height;
+    if (!GetTransaction(hash, tx, Params().GetConsensus(), hashBlock, &height, true))

```

Comment:
don't need height here -> nullptr
as you see, fAllowSlow parameter is changing pretty often, so it would make sense to move pnBlockHeight parameter after fAllowSlow.
This way - you don't need to change so many places, just set default value in main.h file:
   bool fAllowSlow = false, uint32_t *pnBlockHeight = nullptr, ....